gradle/libs.versions.toml

```diff
@@ -38,7 +37,6 @@ aws-sdk2-eventbridge = { group = 'software.amazon.awssdk', name = 'eventbridge',
 aws-sdk2-s3 = { group = 'software.amazon.awssdk', name = 's3', version.ref = 'awsSdk2' }
 aws-sdk2-sqs = { group = 'software.amazon.awssdk', name = 'sqs', version.ref = 'awsSdk2' }
 aws-sdk2-urlconnection = { group = 'software.amazon.awssdk', name = 'url-connection-client', version.ref = 'awsSdk2' }
-aws-sdk2-lambda-events = { group = 'com.amazonaws', name = 'aws-lambda-java-events', version.ref = 'awsSdk2Events' }
```

Comment:
Is this naming just completely wrong? And what are we now using? SDK1 or 2?

Reply:
as far as I understand aws-lambda-java-events includes both sdk1 and 2 events and there is no dedicated sdk2 version on software.amazon... so maybe not wrong directly but maybe a bit weird cause it also includes sdk1 and is on the old namespace?